components/console-backend-service/internal/domain/k8s/resourcequota_service.go

```diff
@@ -47,16 +48,26 @@ func (svc *resourceQuotaService) ListResourceQuotas(namespace string) ([]*v1.Res
 	return result, nil
 }
 
-func (svc *resourceQuotaService) CreateResourceQuota(namespace string, name string, memoryLimits string, memoryRequests string) (*v1.ResourceQuota, error) {
+func (svc *resourceQuotaService) CreateResourceQuota(namespace string, name string, resourceQuotaInput gqlschema.ResourceQuotaInput) (*v1.ResourceQuota, error) {
+	memoryLimitsParsed, err := resource.ParseQuantity(*resourceQuotaInput.Limits.Memory)
+	if err != nil {
+		return nil, errors.Wrap(err, "while parsing resource quota memory limits")
+	}
+
+	memoryRequestsParsed, err := resource.ParseQuantity(*resourceQuotaInput.Requests.Memory)
+	if err != nil {
+		return nil, errors.Wrap(err, "while parsing resource quota memory requests")
```

Comment:
Can you create in pretty package values for ResourceQuota and use in `errors.Wrap`? For example: 
```go
return nil, errors.Wrapf(err, "while parsing %s memory limits", pretty.ResourceQuota)
```